back/spec/factories/custom_fields.rb

```diff
@@ -4,7 +4,8 @@
   factory :custom_field, aliases: [:custom_field_text] do
     for_registration
 
-    sequence(:key) { |n| "field_#{n}" }
+    key { |_n| "field_#{rand(1_000_000..9_999_999)}" }
```

Comment:
This sequence thing was breaking other tests because we are now creating many more custom fields. I didn't really understand what was happening but changing it to a huge randomly generated number solved the issue.